internal/rpcgateway/rpcgateway.go

```diff
@@ -42,14 +44,33 @@ func (r *RPCGateway) Start(ctx context.Context) error {
 			zap.L().Fatal("failed to start healthcheck manager", zap.Error(err))
 		}
 	}()
+	portNumber, err := strconv.Atoi(r.config.Proxy.Port)
+	if err != nil {
+		zap.L().Error("Failed parse port number", zap.Error(err))
+	}
+
+	go func () error {
+		wsListenAddress := fmt.Sprintf(":%d", portNumber + 1)
+
+		zap.L().Info("starting ws failover proxy", zap.String("wsListenAddress", wsListenAddress))
+		listener, err := net.Listen("tcp", wsListenAddress)
+		if err != nil {
+			zap.L().Error("Failed to listen ws", zap.Error(err))
+		}
+		wsListener := conntrack.NewListener(listener, conntrack.TrackWithTracing())
+		conntrack.NewListener(listener, conntrack.TrackWithTracing())
+		return r.wsServer.Serve(wsListener)
```

Comment:
I'm worried that here if ws listener fails the server will continue working. May be instead of returning error (which noone receives by the way), we should panic and let the server to be restarted?